packages/framework/libraries/core/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import { Pipe } from './pipe/model/Pipe';
+import { PipeMetadata } from './pipe/model/PipeMetadata';
+import { isPipe } from './pipe/typeguard/isPipe';
+
+export { isPipe, Pipe, PipeMetadata };
```

Comment:
```suggestion
export { isPipe, type Pipe, type PipeMetadata };
```
Some bundlers find this extra piece of information really useful